docs/web_reference.rst

```diff
@@ -2773,6 +2775,15 @@ application on specific TCP or Unix socket, e.g.::
 
                        ``128`` by default.
 
+.. exception:: GracefulExit
+
+   Raised by classes which inherit from :class:`BaseRunner` whenever
```

Comment:
I don't like that it says that classes raise exceptions. They are containers for stuff, the classes don't do anything short of existing. Methods could raise exceptions but not classes.
Any chance to rephrase this?

Reply:
Hmm, agreed, I used the wrong words here. How about "Raised by signal handlers defined in classes which inherit from :class:`BaseRunner` whenever the ``handle_signals`` parameter is set to ``True``."?

Reply:
Or maybe we should skip the abstraction of "classes which inherit from `BaseRunner`" and just specify that this applies to `AppRunner` and `ServerRunner`? It would be more explicit this way.

Reply:
I suppose we could try the latter.